it/pom.xml

```diff
@@ -21,7 +21,7 @@
     <dependency>
       <groupId>org.sonarsource.orchestrator</groupId>
       <artifactId>sonar-orchestrator</artifactId>
-      <version>3.12</version>
+      <version>3.13</version>
```

Comment:
IT tests need to be refactored to be able to upgrade to 3.14.0.887 (which is an other story IMO)